SQL/proc_descriptions/test_assign_average_speed_to_all_segments_in_area.sql

```diff
@@ -0,0 +1,193 @@
+-- This file is purposed to test next procedure in the database
+-- procedure name: assign_average_speed_to_all_segments_in_area
+-- short name : aastas
+-- test that after execution of this procedure,
+-- some records were added to `nodes_ways_speeds` with corresponding values (which would be hardcoded to the assertion). Status: `approved`.
+-- Note: multiple such tests needed,
+-- 1) `testing both segments with matching speeds,
+-- 2) without them
+-- 3) and the combination of segments with
+-- 4) and without matching speeds in one way`
+-- Note: A segment is a line between two points in a way.
+
+-- begin named transaction
+BEGIN TRANSACTION;
+CALL test_env_constructor();
+SELECT * FROM startup_aastas();
+CALL test_env_destructor();
+END TRANSACTION;
+SELECT * FROM areas;
+SHOW SEARCH_PATH;
+
+-- startup function for aastas
+CREATE OR REPLACE FUNCTION startup_aastas() RETURNS VOID AS $$
+BEGIN
+    RAISE NOTICE 'Executing startup for assign_average_speed_to_all_segments_in_area';
+    RAISE NOTICE 'Inserting into areas';
+    -- insert one area
+    INSERT INTO areas (id, name, description, geom) VALUES (1, 'Test Area', 'Test Area Description', ST_GeomFromText('POLYGON((0 0, 0 10, 10 10, 10 0, 0 0))', 4326));
+
+    RAISE NOTICE 'Inserting into nodes';
+    -- insert 5 nodes within the test area
+    INSERT INTO nodes(id, geom, area, contracted) VALUES (1, ST_GeomFromText('POINT(1 1)', 4326), 1, false)
+    , (2, ST_GeomFromText('POINT(1 5)', 4326), 1, false)
+    , (3, ST_GeomFromText('POINT(5 5)', 4326), 1, false)
+    , (4, ST_GeomFromText('POINT(5 1)', 4326), 1, false)
+    , (5, ST_GeomFromText('POINT(9 1)', 4326), 1, false);
+    -- insert 3 nodes outside the test area
+    -- TODO: i've inserted 2 nodes outside test area and assigned to the test area, maybe we should add CHECK constraint with ST_GeomIntersects to the nodes table,
+    --  because later in this test, these nodes are included in calculations, which is not logical at all.
```

Comment:
@F-I-D-O , also check this suggestion

Reply:
we can trust that nodes will be inside the area identified by area_id